backend/src/users/users.service.ts

```diff
@@ -33,6 +36,7 @@ export class UsersService {
         ...(filter && {
           where: WHERE_CLAUSE,
         }),
+        ...orderBy,
```

Comment:
We want to pass orderBy only if there is actual sorting.
Consider check if there are any sorting exist

Reply:
It will work also like this, I checked it.
But if you want I I will change it.
No problem =)

Reply:
You are correct, but do we really want to send unnecessary objects to our query everytime?

Reply:
done